notification-lambda.js

```diff
@@ -91,26 +73,25 @@ function postNotificationData(notificationData) {
       });
 }
 
-exports.handler =  async function (event, context) {
-    console.log("EVENT: \n" + JSON.stringify(event, null, 2))
-
-    const lastUpdatedTimestamp = await getLastUpdated()
+exports.handler =  async function (event, _) {
 
-    const notificationData = await getNotificationData(lastUpdatedTimestamp)
+    const lastUpdatedTimestamp = await getLastUpdated();
 
-    console.log("Got the notification data LETSSS GOOO: " + notificationData)
+    const notificationData = await getNotificationData(lastUpdatedTimestamp);
 
     if (process.env.SendingEnabled === "true") {
-        await postNotificationData(notificationData).then(response => {
+        try {
+            const response = await postNotificationData(notificationData);
             if (response === 200) {
-                console.log("NOTIFICATION DATA SENT SUCCESSFULLY")
+                console.log("Notification data sent successfully")
             } else {
-                console.log("Failed to send notification, API returned: " + response)
+                throw new Error("Failed to send notification, API returned: " + response)
```

Comment:
```suggestion
                console.log("Failed to send notification, API returned: " + response)
```

I realised that a 400 is going to be an expected response for this (as we will sending notifications with the same id to the notification API and use that to filter out duplicates), so we shouldn't throw here after all...

Reply:
Actually, perhaps we should treat the 400 as a special case - it'd be good to know if we were receiving 5XXs or 401s.